### DISCUSSION TITLE
Collecting links to relevant websites

### DISCUSSION THREAD

**Discussion by GeorgeKerscher** [daisy/math-a11y#31] [Announcements]
This group intends tocollect references to good work. This issue can serve as a mechanism for  collecting those links.

We can use markdown to identify the site and its link. We can also put in a short description.

**Comment by GeorgeKerscher** [comment]
MathML and LaTex Symbols

[List of LaTeX mathematical symbols](https://oeis.org/wiki/List_of_LaTeX_mathematical_symbols)

There are 13 tables that show the MathML and Latex code. This is an interesting grouping.

This site lays them out

**Comment by brichwin** [comment]
Should this issue be in the discussions instead?

**Comment by NSoiffer** [comment]
The strength of TeX is that it is extendable. It is also it's problem. There are various macro packages that define their own names for symbols. AMS LaTeX is one such package. Here is a[ link to a PDF that shows the AMS symbol definitions](https://www.google.com/url?sa=t&source=web&rct=j&opi=89978449&url=https://authortools.aas.org/aastex/aassymbols.pdf).

**Reply by brichwin** [reply]
And in case you can't quite find what you are looking for, try [The Comprehensive LaTeX Symbol List, Scott Pakin, 2024-Jan-03](https://us.mirrors.cicku.me/ctan/info/symbols/comprehensive/symbols-letter.pdf) which at 480+ pages lists 20323 symbols and the corresponding LaTeX commands to produce them.

**Comment by brichwin** [comment]
Here is the Unicode Technical Note for [UnicodeMath Version 3.2 from 13-Apr-23](https://www.unicode.org/notes/tn28/UTN28-PlainTextMath-v3.2.pdf). It's very handy for understanding how UnicodeMath works, the reasoning behind it, the formal syntax, etc.

**Comment by brichwin** [comment]
Murray Sargent contributes to the UnicodeMathML repo and has added an autocomplete listbox feature. He describes it in his [Math Autocomplete blog post](https://learn.microsoft.com/en-us/archive/blogs/murrays/math-autocomplete) You can also experiment with the feature in the: [UnicodeMathML Playground](https://murrayiii.github.io/UnicodeMathML/playground/). In the blog post, Murray describes the feature like so:
> Users of OfficeMath note that entering math symbols using a keyboard can be hard because they don't know the symbol keywords to type and/or the control words are long. To ease these problems, we create a math autocomplete listbox that displays the control words that match what the user types. The user can then use ↑↓ and Enter, mouse or touch to enter the desired symbols. For example, \in<Enter> inserts ∞. You might not know that the TeX control word for ∞ is \infty, rather than \infinity, but you don’t have to know this with the autocomplete listbox.

**Comment by brichwin** [comment]
This would have been useful to have a while ago: [Symbols defined by unicode-math, Will Robertson, 2023/08/13 0.8r](https://ctan.math.utah.edu/ctan/tex-archive/macros/unicodetex/latex/unicode-math/unimath-symbols.pdf)

**Comment by brichwin** [comment]
Writeful.com parsed 300K LaTeX papers on arXiv to explore what LaTeX commands are used most in research writing. They published the results as [These are the 100 most frequent LaTeX commands](https://blog.writefull.com/the-100-most-frequent-latex-commands/). What is useful, is that they tracked which LaTeX commands out of those top 100 were used in math mode. See the article for the full list. Here are just the math mode ones:

4. **\frac (4.9%)** - \frac{numerator}{denominator} produces a fraction in math environments.

9. **\right (3.1%)** - \right* (where * is a delimiter) must be paired with '\left*' (not necessarily
    using the same delimiter). '.' acts as a null delimiter (math mode).

10. **\left (3.1%)** - \left* (where * is a delimiter) must be paired with '\right*' (not necessarily
    using the same delimiter). '.' acts as a null delimiter (math mode).

12. **\alpha (1.6%)** - \alpha is α (math mode).

13. **\mu (1.6%)** - \mu is µ (math mode).

17. **\pi (1.4%)** - \pi is π. \Pi is Π (math mode).

18. **\sigma (1.1%)** - \sigma is σ. \Sigma is Σ (math mode).

19. **\sum (1.1%)** - \sum is ∑ (math mode).

20. **\lambda (1.0%)** - \lambda is λ. \Lambda is Λ (math mode).

21. **\beta (1.0%)** - \beta is β (math mode).

22. **\nu (0.9%)** - \nu is ν (math mode).

23. **\partial (0.9%)** - \partial is ∂ (math mode).

24. **\int (0.9%)** - \int is ∫ (math mode).

25. **\delta (0.9%)** - \delta is δ. \Delta is ∆ (math mode).

26. **\rho (0.9%)** - \rho is ρ (math mode).

27. **\phi (0.9%)** - \phi is φ. \Phi is Φ (math mode).

28. **\gamma (0.9%)** - \gamma is γ. \Gamma is Γ (math mode).

29. **\omega (0.9%)** - \omega is ω. \Omega is Ω (math mode).

31. **\over (0.9%)** - \over can be used to produce fractions in math environments.

33. **\nonumber (0.8%)** - \nonumber is used in an eqnarray environment to suppress equation
    numbering.

34. **\bar (0.8%)** - \bar puts a macron over a letter (math mode).

35. **\sqrt (0.8%)** - \sqrt produces a squared root, for example, \sqrt[3]{arg}. The root ([3]) is
    optional.

36. **\theta (0.8%)** - \theta is θ. \Theta is Θ (math mode).

37. **\tau (0.8%)** - \tau is τ (math mode).

39. **\rangle (0.8%)** - \rangle is ⟩ (math mode).

40. **\hat (0.7%)** - \hat makes a circumflex: ˆ (math mode).

41. **\tilde (0.7%)** - \tilde makes a tilde: ˜ (math mode).

42. **\cal (0.7%)** - \cal produces calligraphic letters (math mode).

44. **\hline (0.7%)** - \hline draws a horizontal line across all columns of a tabular or array
    environment.

47. **\psi (0.6%)** - \psi is ψ. \Psi is Ψ (math mode).

49. **\vec (0.6%)** - \vec puts a vector over a letter (math mode).

50. **\langle (0.6%)** - \langle is ⟨ (math mode).

51. **\epsilon (0.6%)** - \epsilon is ϵ (math mode).

53. **\eta (0.5%)** - \eta is η (math mode).

55. **\cdot (0.5%)** - \cdot is · (math mode).

56. **\in (0.5%)** - \in is ∈ (math mode).

57. **\xi (0.5%)** - \xi is ξ. \Xi is Ξ (math mode).

58. **\infty (0.5%)** - \infty is ∞ (math mode).

63. **\times (0.5%)** - \times is × (math mode).

66. **\mathbf (0.4%)** - \mathbf is used to present math in bold font.

67. **\prime (0.4%)** - \prime is <em>′</em> (math mode).

68. **\be (0.4%) ** - \be is used as an abbreviation for '\begin{equation}'.

69. **\mathrm (0.4%)** - \mathrm is used to turn on roman typestyle for uppercase and lowercase
    letters.

70. **\ee (0.4%)** - \ee is used as an abbreviation for '\end{equation}'.

72. **\pm (0.4%)** - \pm is ± (math mode).

73. **\chi (0.4%)** - \chi is χ (math mode).

76. **\ell (0.3%)** - \ell is ℓ (math mode).

77. **\text (0.3%)** - \text is used to add normal text in math mode.

80. **\to (0.3%)** - \to is → (math mode).

81. **\varphi (0.3%)** - \varphi is ϕ (math mode).

83. **\leq (0.3%)** - \leq is ≤ (math mode).

84. **\cos (0.3%)** - \cos is cos (math mode).

86. **\overline (0.3%)** - \overline{text} produces an overline on 'text' (math mode).

87. **\sin (0.3%)** - \sin is sin (math mode).

88. **\kappa (0.3%)** - \kappa is κ (math mode).

90. **\rightarrow (0.3%)** - \rightarrow is →. \Rightarrow is ⇒ (math mode).

91. **\varepsilon (0.3%)** - \varepsilon is ε (math mode).

93. **\dagger (0.3%)** - \dagger is † (math mode).

95. **\big (0.3%)** - \big is used to set a (larger) size for brackets and parentheses.

96. **\otimes (0.2%)** - \otimes is ⊗ (math mode).

97. **\equiv (0.2%)** - \equiv is ≡ (math mode).

98. **\zeta (0.2%)** - \zeta is ζ (math mode).

99. **\dot (0.2%)** - \dot makes a dot over a letter: ˙ (math mode).

100. **\ln (0.2%)** - \ln is ln (math mode).
